tools/runners.mk

```diff
@@ -21,6 +21,13 @@ $(INSTALL_DIR)/bin/yosys:
 	$(MAKE) -C $(RDIR)/yosys ENABLE_TCL=0 ENABLE_ABC=0 ENABLE_GLOB=0 ENABLE_PLUGINS=0 ENABLE_READLINE=0 ENABLE_COVER=0
 	install -D $(RDIR)/yosys/yosys $@
 
+# yosys-sv
+yosys-sv: $(INSTALL_DIR)/bin/yosys-sv
+
+$(INSTALL_DIR)/bin/yosys-sv:
+	$(MAKE) -C $(RDIR)/yosys-sv ENABLE_TCL=0 ENABLE_ABC=0 ENABLE_GLOB=0 ENABLE_PLUGINS=0 ENABLE_READLINE=0 ENABLE_COVER=0
```

Comment:
Could we only add the `third_party/cores/ibex-yosys-build` submodule and build yosys from the submodule there? I think there is no point in keeping it as our submodule directly